modules/gdnative/gdnative.cpp

```diff
@@ -858,12 +868,56 @@ bool GDNativeInstance::get(const StringName &p_name, Variant &r_ret) const {
 		r_ret = *(Variant *)&value;
 		return true;
 	}
+
+	Map<StringName, GDNativeScriptData::Method>::Element *E = script->script_data->methods.find("_get");
+	if (E) {
+		Variant name = p_name;
+		const Variant *args[1] = { &name };
+
+		godot_variant result = E->get().method.method((godot_object *)owner, E->get().method.method_data, userdata, 1, (godot_variant**) args);
+		r_ret = *(Variant *)&result;
+		return true;
+	}
+
 	return false;
 }
 
 void GDNativeInstance::get_property_list(List<PropertyInfo> *p_properties) const {
 	script->get_script_property_list(p_properties);
 	// TODO: dynamic properties
+
+	Map<StringName, GDNativeScriptData::Method>::Element *E =  script->script_data->methods.find("_get_property_list");
+	if (E) {
+		godot_variant result = E->get().method.method((godot_object *)owner, E->get().method.method_data, userdata, 0, NULL);
+		Variant ret = *(Variant *)&result;
+		Array arr = ret;
+
+		for (int i = 0; i < arr.size (); i++) {
+			Dictionary d = arr[i];
```

Comment:
People should know what is happening with just knowledge of variable and function names.